faraday/proxy.py

```diff
@@ -393,6 +428,208 @@ def testdb_read_worker():
         row = cursor.fetchone()
 
 
+def acceptConnection(server, faraday):
+    conn, addr = server.accept()
+    logger.info("Got connection from {0} on {1}".format(addr, faraday))
+    return conn, addr
+
+
+def closeConnection(conn, addr, faraday):
+    # close the connection
+    logger.info("Closing connection with {0} on {1}".format(addr, faraday))
+    try:
+        conn.close()
+    except socket.error as e:
+        logger.error(e)
+
+
+def extractBytes(data, dataBuffer, unit):
```

Comment:
Is this really possible? That doesn't seem right... Encoding relies on stuffing multiple bytes for single bytes as needed...